packages/react-composites/src/composites/common/SurveyPane.tsx

```diff
@@ -9,30 +9,134 @@ import { useLocale } from '../localization';
 /* @conditional-compile-remove(end-of-call-survey) */
 import { CallSurvey, CallSurveyResponse } from '@azure/communication-calling';
 /* @conditional-compile-remove(end-of-call-survey) */
-import { Panel, PanelType } from '@fluentui/react';
+import { Panel, PanelType, PrimaryButton, Stack } from '@fluentui/react';
 /* @conditional-compile-remove(end-of-call-survey) */
 import { SurveyPaneContent } from './SurveyPaneContent';
+/* @conditional-compile-remove(end-of-call-survey) */
+import { CallSurveyImprovementSuggestions } from '@internal/react-components';
 
 /** @private */
 export const SurveyPane = (props: {
   /* @conditional-compile-remove(end-of-call-survey) */
   onSubmitSurvey?: (survey: CallSurvey) => Promise<CallSurveyResponse | undefined>;
+  /* @conditional-compile-remove(end-of-call-survey) */
+  onSurveySubmittedCustom?: (
```

Comment:
what is the difference between onSubmitSurvey and onSurveySubmittedCustom?

Reply:
Can they be rolled into one prop?

Reply:
onSubmitSurvey comes from calling directly and onSurveySubmittedCustom is the function contoso pass in to get access to survey data. They take different props so can't be rolled into one